### PR TITLE
[VEN-2090][VEN-2188]: XVS bridge src chain configurations for Sepolia and Ethereum

### PR DESCRIPTION
**Pull request by GitGuru7**
## Description

Resolves VEN-2090, VEN-2188

### CODE REVIEW

File: vips/vip-187/vip-187-testnet-3.ts
Comment (chechu) [inline vips/vip-187/vip-187-testnet-3.ts:1]: Do we need the previous VIP files?

Comment (GitGuru7) [reply vips/vip-187/vip-187-testnet-3.ts:1]: No, I just wanted to keep it easier for review, Anyway removed here [46ddb97](https://github.com/VenusProtocol/vips/pull/140/commits/46ddb9755f01c42b18b372c1e55fbb2419e8a033)

**Comment by GitGuru7** [comment]
https://testnet.bscscan.com/tx/0x9e9834db9782b33d3193730dfcef105ddda3bcaf5da12fa61be0583c93f25f36

File: vips/vip-187/vip-187-testnet.ts [hunk 1]
```diff
@@ -128,7 +143,7 @@ export const vip187Testnet = () => {
         signature: "giveCallPermission(address,string,address)",
         params: [XVSBridgeAdmin_Proxy, "transferBridgeOwnership(address)", NORMAL_TIMELOCK],
       },
-      // FASTTRACK TIMELOCK PERMISSIONS
+      //  FASTTRACK TIMELOCK PERMISSIONS
```
Comment (chechu) [inline vips/vip-187/vip-187-testnet.ts:146]:
```suggestion
      // FASTTRACK TIMELOCK PERMISSIONS
```

File: vips/vip-187/vip-187-testnet.ts [hunk 2]
```diff
@@ -128,7 +143,7 @@ export const vip187Testnet = () => {
...
       {
```
Comment (chechu) [inline vips/vip-187/vip-187-testnet.ts:147]: I think these functions should be executable only by the Normal timelock, so I wouldn't grant Fast-track and Critical timelocks to execute them:
* setPrecrime(address)
* setTrustedRemoteAddress(uint16,bytes)
* transferBridgeOwnership(address)

**Review by chechu — COMMENTED** [review]
Sepolia VIP lgtm

**Review by chechu — APPROVED** [review]

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/232?chainId=56

**Pull request merged**